.github/workflows/terraform.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Lokomotive Unit Tests'
```

Comment:
Why is this part of this PR? We use Concourse for our CI. I'm not sure if we should be mixing multiple CI systems for our checks.